src/consumption/elevate.js

```
@@ -433,6 +433,450 @@ async function createTasks(tasks, templateId, templateExternalId, parentId = nul
+	} catch (error) {
+		console.error('Error in formatTemplate:', error.message)
+		return { success: false, error: error.message }
+	}
+}
+/**
+ * Publish the project template
+ * @name publishProjectTemplates
+ * @param {Object} templateData - Project template data
+ * @returns {Object} - Response of template creation
+ */
+const publishProjectTemplates = function (templateData) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'publishProjectTemplates'. Consider to delete it or rename it.

'publishProjectTemplates' is defined here:


(lint/suspicious/noRedeclare)